server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetVacationResponseTest.java

```diff
@@ -72,12 +79,112 @@ public void teardown() {
     }
 
     @Test
-    public void getVacationResponseIsNotImplementedYet() {
+    public void getVacationResponseShouldReturnDefaultValue() {
+        given()
+            .accept(ContentType.JSON)
+            .contentType(ContentType.JSON)
+            .header("Authorization", accessToken.serialize())
+            .body("[[" +
+                "\"getVacationResponse\", " +
+                "{}, " +
+                "\"#0\"" +
+                "]]")
+        .when()
+            .post("/jmap")
+        .then()
+            .statusCode(200)
+            .body(NAME, equalTo("vacationResponse"))
+            .body(ARGUMENTS + ".accountId", equalTo(username))
```

Comment:
The spec says on the vacationResponse : 

```
accountId: String The id of the account used for the call.
```

Different from the request which is : 

```
accountId: String|null The Account to get the vacation response for. If null, the primary account is used.
```

I understand *answer the accountId truely used* :-)